airflow/www/templates/airflow/trigger.html

```diff
@@ -72,7 +72,7 @@
     {% elif "enum" in form_details.schema and form_details.schema.enum %}
       <select class="my_select2 form-control" name="element_{{ form_key }}" id="element_{{ form_key }}" data-placeholder="Select Value"
         onchange="updateJSONconf();"
-        {%- if "integer" in form_details.schema.type %} valuetype="number" {% elif "number" in form_details.schema.type %} valuetype="decimal" {%- endif %}
+        {%- if "integer" in form_details.schema.type or "number" in form_details.schema.type %} valuetype="number"{% endif %}
```

Comment:
So just to make sure, this PR reverts this line from #31946 and the aggregated change is only line 120, right?

Reply:
Yes, because we need just to change `type` attribute